packages/type-safe-api/src/project/codegen/runtime/generated-typescript-async-runtime-project.ts

```diff
@@ -16,18 +27,74 @@ export interface GeneratedTypescriptAsyncRuntimeProjectOptions
  * Typescript project containing types generated using OpenAPI Generator CLI
  */
 export class GeneratedTypescriptAsyncRuntimeProject extends GeneratedTypescriptRuntimeBaseProject {
+  // TODO: remove
+  /**
+   * Patterns that are excluded from code generation
+   */
+  public static openApiIgnorePatterns: string[] = [
+    "package.json",
+    "tsconfig.json",
+    "tsconfig.esm.json",
+    ".npmignore",
+  ];
+
+  protected readonly openapiGeneratorIgnore: OpenApiGeneratorIgnoreFile;
+
   constructor(options: GeneratedTypescriptAsyncRuntimeProjectOptions) {
     super(options);
 
     this.addDeps("@aws-sdk/client-apigatewaymanagementapi");
 
-    this.openapiGeneratorIgnore.addPatterns(
+    // TODO: remove this in favour of new codegen
```

Comment:
Any reason we can't do this now?

Reply:
I just haven't written the ejs templates yet - will do as a follow up PR :)